### ISSUE TITLE
Title of notebook changes to shortened form

### ISSUE THREAD

**Issue by Iznik285** [dvorka/mindforger#1318]
On Windows 10 / MF 1.52.0 when entering the first notebook in a session it is displayed in a shortened form 
e.g. 
![image](https://user-images.githubusercontent.com/63961864/91614307-39e75d00-e979-11ea-811c-aeb56b5a92fe.png)

When you look at any subsequent notebooks they are all in the longer form.

**Comment by dvorka** [comment]
@Iznik285 yes, it's still reproducible, however, I don't know how to fix it yet. Anyway I really appreciate your bug report. Hopefully I find a solution soon.
